src/frontend/src/components/Login/LoggedIn.jsx

```diff
@@ -32,12 +33,16 @@ function LoggedIn(props) {
           Logout
         </a>
       </Button>
-      <Typography variant="h4">| Welcome {props.name}</Typography>
-      <img
-        className={classes.avatar}
-        src={`https://unavatar.now.sh/${props.name}`}
-        alt={props.email}
-      />
+      <Link to="/myfeeds" className={classes.links}>
+        <Typography variant="h4">| Welcome {props.name}</Typography>
```

Comment:
Let's drop `Welcome `, it's too long

Reply:
Done in https://github.com/Seneca-CDOT/telescope/commit/8cab1c2bb69b1fd64b6cc7f3ab73404746dabbef